### PR TITLE
Fix price column size

### PR DESCRIPTION
**Pull request by Hlavtox**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Prince index column was too small and sorting did not work for big numbers.
| Type?         | bug fix
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes PrestaShop/Prestashop#25789
| How to test?  | Make products with prices higher than 999999 and test that sorting now works correctly.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/ps_facetedsearch/506)

### CODE REVIEW

File: upgrade/upgrade-3.8.0.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+if (!defined('_PS_VERSION_')) {
+    exit;
+}
+
+function upgrade_module_3_8_0($module)
+{
+    return Db::getInstance()->execute(
+      'ALTER TABLE `' . _DB_PREFIX_ . 'layered_price_index` 
+      CHANGE `price_min` `price_min` decimal(20,6) NOT NULL AFTER `id_shop`,
+      CHANGE `price_max` `price_max` decimal(20,6) NOT NULL AFTER `price_min`;');
```
Comment (PierreRambaud) [inline upgrade/upgrade-3.8.0.php:29]:
```suggestion
      CHANGE `price_min` `price_min` decimal(20,6) NOT NULL,
      CHANGE `price_max` `price_max` decimal(20,6) NOT NULL;');
```
I don't think move columns is needed

Comment (Hlavtox) [reply upgrade/upgrade-3.8.0.php]: Done ;-) I was lazy and copied adminer query :D

**Review by PierreRambaud — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by florine2623** [comment]
Hello @Hlavtox ,

Thanks for the PR. All good to me !

**Comment by kpodemski** [comment]
@PrestaShop/prestashop-maintainers can I ask for the rights? :) I wanted to merge it, but I can't

**Comment by matks** [comment]
> @PrestaShop/prestashop-maintainers can I ask for the rights? :) I wanted to merge it, but I can't

Weird 😮 

Is it better now ? modified the access settings

**Comment by kpodemski** [comment]
@matks yes, it is working now, thank you :)

and thank you @Hlavtox & @florine2623  💪

**Pull request merged**